agimus_controller_ros/sim_utils.py

```diff
@@ -0,0 +1,23 @@
+from std_msgs.msg import Duration
+import numpy as np
+
+
+def convert_float_to_ros_duration_msg(time):
+    ros_duration = Duration()
+    ros_duration.data.secs = int(time)
+    ros_duration.data.nsecs = int((time % 1) * 1e9)
```

Comment:
mmmh... not sure about that...
Would that be more logical?
Did you test it?
```suggestion
    ros_duration.data.nsecs = int(((time - ros_duration.data.secs) % 1) * 1e9)
```

Reply:
doing `time % 1` or `time - ros_duration.data.secs ` is exactly the same, it's less cumbersome this way, I tested it yes

Reply:
ok